evaluate.py

```diff
@@ -89,13 +90,22 @@ def len_filter(example):
     return len(example.src) <= max_len and len(example.tgt) <= max_len
 
 
+def get_standard_batch_iterator(data, batch_size):
```

Comment:
I assume this returns the iterator as it was before. Should we maybe extend this such that you can also get other iterators, such as the one needed for language modelling?